server/http.go

```diff
@@ -196,6 +250,7 @@ func httpWorkflowRegister(p *Plugin, w http.ResponseWriter, r *http.Request) (in
 						Description: "Jira issue ID",
 					},
 				},
+				//TODO <><> prefix route with instance? or unnecessary since it's for all instances?
```

Comment:
Resolve the TODO

Reply:
I created an issue to track and resolve all TODO comments.  https://github.com/mattermost/mattermost-plugin-jira/issues/611